src/Core/App.xaml.cs

```diff
@@ -331,18 +331,27 @@ void NavigateToSetPasswordPageAction()
                     || message.Command == "unlocked"
                     || message.Command == AccountsManagerMessageCommands.ACCOUNT_SWITCH_COMPLETED)
                 {
-                    if (message.Command == AccountsManagerMessageCommands.ACCOUNT_SWITCH_COMPLETED)
+                    var userVerificationMediatorService = ServiceContainer.Resolve<IFido2MakeCredentialConfirmationUserInterface>();
```

Comment:
⛏️ Move it to a field with `LazyResolve` so it's not resolve every time the account switches.